core/src/test/scala/integration/kafka/api/SaslSslAdminIntegrationTest.scala

```diff
@@ -331,6 +332,28 @@ class SaslSslAdminIntegrationTest extends BaseAdminIntegrationTest with SaslSetu
     assertFutureExceptionTypeEquals(results.values.get(emptyResourceNameAcl), classOf[InvalidRequestException])
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = Array("zk", "kraft"))
+  def testCreateDelegationTokenWithOptions(quorum: String): Unit = {
+    val owner = new KafkaPrincipal("User", "client2")
+    val renewers = List(new KafkaPrincipal("User", "client")).asJava
+    client = createAdminClient
+    val timeout = 1000
+    val options = new CreateDelegationTokenOptions().maxlifeTimeMs(timeout).owner(owner).renewers(renewers)
```

Comment:
this is duplicate to #16648. this PR should test "delegation.token.expiry.time.ms" and "delegation.token.max.lifetime.ms". For example, this test case should set negative `maxlifeTimeMs` and larger `maxlifeTimeMs` to make sure `maxTimestamp` and `expiryTimestamp` does not exceed the configured limits.